packages/lb-components/src/components/pointCloudView/hooks/usePointCloudViews.ts

```diff
@@ -809,27 +809,36 @@ export const usePointCloudViews = () => {
       boxID: string;
       imageName: string;
     },
+    action: 'remove' | 'update' = 'update',
```

Comment:
how about implement update/remove, this will may update2DViewRect more complex

Reply:
Using the new method creates a lot of duplicate code